projects/Apps/editionsSrc/src/authentication/services/iap.ts

```diff
@@ -56,7 +55,7 @@ const isReceiptValid = (receipt: ReceiptIOS) => {
     return expirationWithGracePeriod > nowInMilliseconds
 }
 
-const findValidReceipt = (receipt: ReceiptValidationResponse) =>
+const findValidReceipt = (receipt: any) =>
```

Comment:
Why has this needed to change?

Reply:
Typescript says it's not exported, which is correct.